tests/js/spec/utils/useLocation.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {createMemoryHistory, Route, Router, RouterContext} from 'react-router';
+
+import {render} from 'sentry-test/reactTestingLibrary';
+
+import useLocation from 'sentry/utils/useLocation';
+import {RouteContext} from 'sentry/views/routeContext';
+
+describe('useLocation', () => {
+  it('returns the current location object', function () {
+    let location;
+    function HomePage() {
+      location = useLocation();
+      return null;
+    }
+
+    const memoryHistory = createMemoryHistory();
+    memoryHistory.push('/?hello');
+
+    render(
+      <Router
+        history={memoryHistory}
+        render={props => {
+          return (
+            <RouteContext.Provider value={props}>
+              <RouterContext {...props} />
+            </RouteContext.Provider>
+          );
+        }}
+      >
+        <Route path="/" component={HomePage} />
+      </Router>
+    );
+
+    expect(typeof location).toBe('object');
```

Comment:
I think you are ok to remove this too. You could check for instanceof, but idk if that would even work because iirc there is no location constructor